R/RunWorkflow.R

```diff
@@ -90,6 +94,14 @@ RunWorkflow <- function(
 
     lWorkflow$lData[[step$output]] <- result
     lWorkflow$lResult <- result
+
+    if (!is.null(step$save) && step$save) {
```

Comment:
I think we should get rid of step$save. 

Instead, I think we can probably support saving a list of data by adding some basic iteration in SaveData(). Something along the lines of: 

```
if(is.list(lWorkflow$result)){
   lWorkflow$result %>% map(SaveData)
} else if(is.data.frame(lWorkflowResult)) {
 #existing code goes here
} else {
    cli_warning("data format not supported, not saving")
```

Reply:
So does this mean we are putting the last step of the metric yamls back in- where we write all tables out as a list? In the current version, this step has been removed.

Reply:
Alternatively, we just set `bReturnResult` to FALSE when running locally and keep the `save` field, but add some logic so that it only triggers a `SaveData()` when `lConfig` is not null.